gitlab_test.go

```diff
@@ -0,0 +1,266 @@
+package reviewdog
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/xanzy/go-gitlab"
+	"net/http"
```

Comment:
please run 'goimports'. imports of standard library should be grouped.

Reply:
done